test/guest/lib/zero-disk.setup

```diff
@@ -18,9 +18,9 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
 
-# on Debian we don't want to delete pbuilder
+# on Debian we don't want to delete pbuilder or cracklib
 if [ -d "/var/cache" ]; then
-    ( cd /var/cache; ls . | grep -vx pbuilder | xargs rm -rf )
+    ( cd /var/cache; ls . | grep -vx "pbuilder\|cracklib" | xargs rm -rf )
```

Comment:
> The command above breaks with paths with spaces in their names. But if that's not the case, lets just leave it. To use find, I think one would use the -maxdepth and -prune arguments, but we can skip that for now.

Since it seems to work for now, this can be a follow-up.